### ISSUE TITLE
Implicit_wait is no longer working on selenium/node-firefox 3.8.0

### ISSUE THREAD

**Issue by ylavoie** [teodesian/Selenium-Remote-Driver#350]
It seems that implicit_wait was removed from selenium/node-firefox, thus preventing the driver initialization.
```
Error while executing command: setImplicitWaitTimeout: POST /session/4cb98b2c-dff8-4f69-b407-32b1f28dc3f2/timeouts/implicit_wait did not match a known command at /usr/local/share/perl/5.22.1/Selenium/Remote/Driver.pm line 327.
 at /usr/local/share/perl/5.22.1/Selenium/Remote/Driver.pm line 327.
# Looks like your test exited with 255 before it could output anything.
```

**Comment by teodesian** [comment]
That sounds pretty serious.  I wonder what it's been replaced with -- I'll have to look that up and put in a shim to not break everybody's everything.

**Comment by teodesian** [comment]
`POST | /session/{session id}/timeouts | Set Timeouts`

Seems we now have to specify what type of timeout it is:

`"script" | session script timeout | Determines when to interrupt a script that is being    evaluated.`
`-- | -- | --`
`"pageLoad" | session page load timeout | Provides the timeout limit used to interrupt   navigation of the browsing context.`
`"implicit" | session implicit wait timeout | Gives the timeout of when to abort    locating an element.`

**Comment by ylavoie** [comment]
I posted an issue on Selenium about node-Firefox too.

**Comment by sbts** [comment]
@ylavoie can you reference the Selenium issue here please

**Comment by teodesian** [comment]
Likely dupe of #345, leaving open until done there "just to be sure"

**Comment by ylavoie** [comment]
Most likely, yes

**Comment by gerhardj** [comment]
Should probably work with, https://github.com/teodesian/Selenium-Remote-Driver/commit/f9296a568d0e4ddc53cc18561d64f98508d74bfa

**Comment by teodesian** [comment]
Have fixed on the #345 branch 'webdriver3'.  Feel free to use @gerhardj's patch in the meantime though -- it looks like it should work fine.
